### PR TITLE
fix: include authorize screen

### PR DESCRIPTION
**Pull request by hvhallmann**

### CODE REVIEW

File: src/pages/oauth/ClientAuthentication.njs [hunk 1]
```diff
@@ -37,25 +45,53 @@ class ClientAuthentication extends Nullstack {
             <strong>E-mail</strong> { me.email }
           </p>
         </div>
+        <br />
```
Comment (jonbertoldi) [inline src/pages/oauth/ClientAuthentication.njs]: Best to apply margin in here instead of `<br>`. You can use `mb-4` in the above div

**Review by jonbertoldi — CHANGES_REQUESTED** [review]

**Comment by hvhallmann** [comment]
fixed

File: src/pages/oauth/ClientAuthentication.njs [hunk 2]
```diff
@@ -37,50 +37,42 @@ class ClientAuthentication extends Nullstack {
...
+                  Permission to read your profile data
+                </label>
               </li>
               <li>
-                <input type="checkbox" name="check_wallet" bind={this.read_wallet_address} >
-                  <label for="check_wallet" class="font-['Poppins'] text-lg m-3">
-                    Permission to read your wallet addresses
-                  </label>
-                </input>
+                <input type="checkbox" name="check_wallet" bind={this.read_wallet_address} />
```
Comment (jonbertoldi) [inline src/pages/oauth/ClientAuthentication.njs:63]: We need to create components here. But lets change when we move to the main repo

**Review by jonbertoldi — COMMENTED** [review]
look good

**Pull request merged**